src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1200px;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.logo-container a {
+  display: block;
+}
+
+.logo-container img {
+  display: block;
+}
+
+.nav ul {
+  display: flex;
+  list-style-type: none;
+}
+
+.nav ul li {
+  position: relative;
+  overflow: hidden;
+}
```

Comment:
```suggestion
```
redundant styles
 .nav__item will be responsible just for margins between other nav items